framework/wazuh/core/authentication.py

```diff
@@ -1,12 +1,62 @@
-from typing import Tuple
+import contextlib
+import os
 
+from cryptography.hazmat.primitives import serialization
+from wazuh.core import common
 from wazuh.core.config.client import CentralizedConfig
+from wazuh.core.config.models.base import WazuhConfigBaseModel
 
-JWT_ALGORITHM = 'ES256'
+JWT_ALGORITHM = 'RS256'
 JWT_ISSUER = 'wazuh'
 
 
-def get_keypair() -> Tuple[str, str]:
+def check_jwt_keys(api_config: WazuhConfigBaseModel):
+    """Verify if JWT key files are configured and generate them if not.
```

Comment:
If the private key doesn't exist, we are not generating it. So we should update this docstring to something like

```suggestion
def load_jwt_keys(api_config: WazuhConfigBaseModel):
    """Load JWT keys into the configuration.
```

Reply:
Changed in 77dfbbd6ab6a16df3f4a0984fbbb3640d6cc1164.